package.json

```diff
@@ -33,29 +34,29 @@
     "dev:fresh": "h2-build && turbo run dev --filter='./apps/*' --filter='!@gc-digital-talent/e2e' --force",
     "dev:project": "turbo run dev",
     "lint": "turbo run lint",
-    "storybook:project": "npm run storybook",
-    "storybook": "npm run storybook --workspace=@gc-digital-talent/web",
-    "storybook:design-system": "npm run storybook:design-system --workspace=@gc-digital-talent/web",
-    "storybook:web": "npm run storybook:web --workspace=@gc-digital-talent/web",
-    "build-storybook": "npm run build-storybook --workspace=@gc-digital-talent/web",
+    "storybook:project": "pnpm run storybook",
```

Comment:
Should this be just `pnpm storybook`?  Same with `e2e:open`?

Reply:
okay, this is actually a very good question. the second arg is assumed to be a script unless another built in command is added.

I did this to make things more simple but honestly, I already don't really like it and think we could explicitly add `run`.

So, we should make it consistent but I'm not sure what pattern to use. Any opinions?

Reply:
I flagged it because you seemed to be going with the other convention.  I'm guessing most of the team is unfamiliar with pnpm and we're using it because it's "drop in" so I'd recommend going with the familiar `pnpm run <script>` form.

Reply:
I agree, updated in [update pnpm calls to avoid assumed run](https://github.com/GCTC-NTGC/gc-digital-talent/pull/9476/commits/19ffbbc79635e7c4bfd24cf3db11f7c7558ef56e)

Reply:
Looks like you're still missing some `run`s in the root package.json?

Reply:
Hmm, I can't seem to find any commands that are missing it. Perhaps the [order of options](https://pnpm.io/cli/run#options) is confusing? 😅 If that isn't it do you mind helping me find the ones that are missing?

Reply:
Oops, sorry, you're right.  I missed the run command after the filter options.  :confounded: